src/test/java/com/shigotosagashi/jobdomainservice/repository/JobEntityRepositoryTest.java

```diff
@@ -0,0 +1,97 @@
+package com.shigotosagashi.jobdomainservice.repository;
+
+import com.shigotosagashi.jobdomainservice.domain.JobEntity;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+import java.time.Instant;
+import java.util.List;
+import java.util.Random;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DataJpaTest
+public class JobEntityRepositoryTest extends BaseRepositoryTest {
+
+  @Autowired
+  private JobEntityRepository repository;
+
+
```

Comment:
🙃

Reply:
?

Reply:
Is the extra line?

Reply:
Yes, that's the extra line. Sorry if I was not clear enough.